src/script/ceph-release-notes

```diff
@@ -138,21 +139,19 @@ def make_release_notes(gh, repo, ref, plaintext, verbose, strict, use_tags):
                     line = line.strip()
                     if line:
                         lines.append(line)
+                if len(lines) == 0:
+                    continue
                 duplicates_pr_title = lines[0] == pr['title'].strip()
                 if duplicates_pr_title:
-                    lines.pop(0)
-                if len(lines) == 0:
-                    if duplicates_pr_title:
-                        message = None
-                elif len(lines) == 1:
+                    continue
+                assert len(lines) > 0, "missing message content"
+                if len(lines) == 1:
```

Comment:
@tchaikov It took me awhile to figure out what this code is actually doing. It's parsing the merge commit message (not the title). First, it removes any empty lines and lines that start with "Reviewed-by". Then, if there is only one (non-empty, non-Reviewed-by) line left, and it is different from the PR title, it replaces the PR title with that line. In all other cases, it appends the (non-empty, non-Reviewed-by) content from the merge commit message to the PR title.

Does my patch make more sense now, in light of the above?